pyproject.toml

```
@@ -88,9 +88,10 @@ test = [
     "joblib",
     "boltons",
     "scanpy",
-    "httpx", # For data downloading
+    "httpx",                   # For data downloading
     "dask[array,distributed]",
     "awkward>=2.3",
+    "pyarrow",                 # For non-deprecated awkward array functionality
```

```suggestion
    "pyarrow",
```

I find the wording of this comment confusing. I would also be fine with no comment since I think pyarrow as a test dependency will be coming for other reasons soon anyways.